docs/_Docs_DomainsSkills.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Domains and Skills
+section: Docs
+order: 5
+---
+
+Domains and Skills are concepts that define a colony's organizational structure in a decentralized context, and allow for the division of labor without a strict management hierarchy.
+
+Domains are a structure for compartmentalizing the work and shared resources of a colony into smaller, more specialized sub-groups which are analogous to the departments of a traditional company.
+
+Skills are a similar structure that categorize the _type_ of work done, independent of the domain or colony in which the work took place.
+
+==TOC==
+
+## Domains
+Like a file system that contains files and sub-folders, any domain can contain tasks, sub-domains, or both.
+
+![Example Domains](https://raw.githubusercontent.com/JoinColony/colonyNetwork/develop/docs/img/domains_r1.svg?sanitize=true)
+
+With the exception of the top-level colony domain, which contains all tasks and sub-domains, every domain in a colony belongs to a larger parent domain.
+
+Every domain has an associated [pot](/colonynetwork/docs-pots-and-funding/) which can recieve funding (in most circumstances from the parent domain's pot).
+
+
+Tasks are tagged with a `DomainId` that specifies which domain they belong to. When a member of a colony earns native tokens for completing a task, they earn reputation in that domain as well as all parent domains, including the colony-wide domain. They do not earn any reputation in child domains.
```

Comment:
Note that not only the worker, but all members who participated in the delivery of a task earn domain reputation, e.g. manager and evaluator as well.